client/src/models/organizational_entities.js

```diff
@@ -201,6 +201,8 @@ const Dept = class Dept extends static_subject_store_with_API_data() {
       } else {
         return "separate_agencies";
       }
+    } else {
+      return undefined;
```

Comment:
Huh, did the new lint rules suggest this?

When execution reaches the end of a function's body without an explicit return, it will return undefined by default.

Reply:
yeah exactly, it was `getter-return`, which is a rule now included in eslint recommended list. I just simply put return undefined to exactly mimic the function and resolve the linter